source/lac/petsc_parallel_vector.cc

```diff
@@ -90,6 +91,23 @@ namespace PETScWrappers
     }
 
 
+
+    void
+    Vector::clear ()
+    {
+      // destroy the PETSc Vec and create an invalid empty vector,
+      // so we can just as well create a sequential one to avoid
+      // all the overhead incurred by parallelism
+      attained_ownership = true;
+      VectorBase:clear ();
```

Comment:
What an interesting problem (found by the indentation script): This compiles, but it doesn't do what you want without adding a second colon :-)